docs/source/_static/css/comp-soft.css

```diff
@@ -0,0 +1,76 @@
+.comp-soft-wrapper {
+
+  width: 700px;
+  margin: 0 auto;
+
+  --s: 150px;  /* size  */
+  --m: 8px;    /* margin */
+  --f: calc(1.732 * var(--s) + 4 * var(--m)  - 1px);
+}
+
+@media (max-width: 767px) {
+  .comp-soft-wrapper {
+
+    width: 100vw;
+    margin: 0 auto;
+  }
+}
+
+.comp-soft-section {
+  width: 100vw;
+  background-color: var(--colorPrimaryDark);
+  padding: 30px 0 90px;
```

Comment:
Is there any reason for increasing the padding at the bottom? I would keep the same 30px so that it looks even.